wrangler-transform/src/e2e-test/features/Wrangler/DataTypeParsers.feature

```diff
@@ -12,10 +12,10 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-@Wrangler
+@Wranglers
```

Comment:
similarly here?

Reply:
I have removed this one in these PR's because this PR is just to check for the avro and log parsers and rest are all already merged. and there are 2 PR's that are open and  in those  i changed the data in some of the files. So just to run avro and log. I just gave a random name. If those get PR's merged than we can rebase it here also.

Reply:
Resolved.